scripts/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
is here some big difference that it couldn't use the same script as it is in project-base with some additional relative path ?
so it would be maintained mainly from one place ?